.github/workflows/aes.yml

```diff
@@ -3,6 +3,7 @@ name: aes
 on:
   pull_request:
     paths:
+      - ".github/workflows/aes.yml"
```

Comment:
We should probably do this everywhere in order to ensure changes to a workflow trigger the workflow